{"name": "test_mail", "status": "broken", "statusDetails": {"message": "NameError: name 'pege' is not defined", "trace": "tests/test_ukr_net.py:9: in test_mail\n    assert \"We want your feedback!\" in pege.find_mail()\nE   NameError: name 'pege' is not defined"}, "attachments": [{"name": "log", "source": "346ccaba-a36f-4bb4-99eb-f0085b0807d4-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "57de370c-e4a0-4e63-800d-91a13a3cbe9d-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "c4764597-2748-475b-a7f2-33a2a7d49622-attachment.txt", "type": "text/plain"}], "start": 1546434990527, "stop": 1546434994915, "uuid": "1c2aea48-24eb-4aa2-a257-73573c66f3ff", "historyId": "2fd3086c7e884ee2689d7c6b26f341dd", "fullName": "tests.test_ukr_net#test_mail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_ukr_net"}, {"name": "host", "value": "MacBook-Pro-Aleksandr.local"}, {"name": "thread", "value": "13633-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_ukr_net"}]}